### PR TITLE
Improve testing workflow

### PR DESCRIPTION
**Pull request by f0rmiga**
## Description

- Improved testing workflow by preventing pushing the built image.
- Improved the cluster creation script.
- Modified the Makefile to allow passing the `imagePullPolicy` as `never`.

## Test plan

Follow the updated README.

### CODE REVIEW

**Comment by zulandar** [comment]
@f0rmiga should minibroker start using kubernetes 1.14? Currently in the `create-cluster.sh` script it set to `v1.11.3`

File: README.md [hunk 1]
```diff
@@ -207,14 +207,16 @@ the database to create, etc.
...
 
-1. Create a Minikube cluster for local development: `make create-cluster`.
-2. Identify where you will push Docker images with your changes and set the `REGISTRY`
-   environment variable. For example to push your dev images to Docker Hub, use
-   `export REGISTRY=myusername/`.
+1. Create a Minikube cluster for local development by running `make create-cluster`. It defaults to
+  using VirtualBox as a VM driver. If you want to use a different VM driver, set the `VM_DRIVER`
+  environment variable. E.g. `VM_DRIVER=kvm2 make create-cluster`.
+2. Point your Docker to use the Minikube Docker daemon on the current shell session by running
+  `eval $(minikube docker-env)`.
```
Comment (carolynvs) [inline README.md]: Nice! This is great, now I don't have to push my images, thank you! 👍

**Comment by carolynvs** [comment]
@f0rmiga oops just saw the merge conflict, once that is fixed, this should be good to go. If we want to bump the version of k8s, that's fine to happen here or in a later PR. Up to you.

File: README.md (hunk 1 above)
Comment (f0rmiga) [reply README.md]: Yay! You are welcome. I love this Minikube feature. :)

**Comment by f0rmiga** [comment]
@zulandar @carolynvs I'll handle the k8s version bump on another PR.

**Comment by f0rmiga** [comment]
@carolynvs With the new push, I dismissed `lgtm`. Please, take another 👀 . Thanks!

**Pull request merged**